src/selfplay/tournament.cc

```diff
@@ -70,7 +70,12 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
   auto defaults = options->GetMutableDefaultsOptions();
   defaults->Set<int>(Search::kMiniBatchSizeStr, 32);     // Minibatch size
   defaults->Set<bool>(Search::kSmartPruningStr, false);  // No smart pruning
-  defaults->Set<float>(Search::kTemperatureStr, 1.0);    // Temperature = 1.0
+  defaults->Set<float>(Search::kMoveTempStr, 1.0);    // Temperature = 1.0
+  defaults->Set<float>(Search::kMoveTempDecayStr, 0.0); // no temp decay
+  //defaults->Set<float>(Search::kMoveTempStr, 1.0);
```

Comment:
What are those lines for?

Reply:
These are my current suggested defaults for selfplay. I guess I can remove them for PR purposes